planning/obstacle_velocity_limiter/src/obstacle_velocity_limiter_node.cpp

```diff
@@ -80,6 +79,18 @@ ObstacleVelocityLimiterNode::ObstacleVelocityLimiterNode(const rclcpp::NodeOptio
 
   set_param_res_ =
     add_on_set_parameters_callback([this](const auto & params) { return onParameter(params); });
+
+  // transform
+  rclcpp::Clock::SharedPtr clock = std::make_shared<rclcpp::Clock>(RCL_ROS_TIME);
+  map_odom_tf_buffer_ = std::make_shared<tf2_ros::Buffer>(clock);
```

Comment:
Also here, can use existing clock with `this->get_clock()` (from `rclcpp::Node`).

Reply:
> Also here, can use existing clock with `this->get_clock()` (from `rclcpp::Node`).

got it. thanks.